__tests__/app.test.js

```diff
@@ -54,6 +54,33 @@ describe("/api/articles", () => {
   });
 });
 
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 - Responds with single article with the correct article_id", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(2);
```

Comment:
You should also check that the object conforms to the expected structure using something like expect.objectContaining or .toMatchObject